### PR TITLE
Do not check certificates for archive.apache.org

### PR DESCRIPTION
**Pull request by wendigo**

### CODE REVIEW

File: testing/phoenix5/Dockerfile [hunk 1]
```diff
@@ -32,7 +32,7 @@ ARG PHOENIX_BINARY_PATH=$BASE_URL/phoenix/phoenix-$PHOENIX_VERSION/phoenix-hbase
 RUN mkdir -p $IMAGE_ROOT_INSTALL
 
 # Download ZooKeeper
-RUN wget --quiet $ZOOKEEPER_BINARY_PATH && \
+RUN wget --no-check-certificate -nv $ZOOKEEPER_BINARY_PATH && \
```
Comment (findepi) [inline testing/phoenix5/Dockerfile:35]: `--no-check-certificate` should be code-documented i guess

**Comment by findepi** [comment]
this sounds wrong. Why do we need this?

**Comment by wendigo** [comment]
@findepi master doesn't build anymore

**Comment by wendigo** [comment]
@findepi
```
 > [2/6] RUN set -xeu;     wget -nv "https://archive.apache.org/dist/spark/spark-3.4.2/spark-3.4.2-bin-hadoop3.tgz";     tar -xf spark-3.4.2-bin-hadoop3.tgz;     rm spark-3.4.2-bin-hadoop3.tgz;     ln -sn /spark-3.4.2-bin-hadoop3 /spark:
0.147 + wget -nv https://archive.apache.org/dist/spark/spark-3.4.2/spark-3.4.2-bin-hadoop3.tgz
0.571 ERROR: cannot verify archive.apache.org's certificate, issued by ‘/C=US/O=Let's Encrypt/CN=R3’:
0.571   Issued certificate has expired.
0.571 To connect to archive.apache.org insecurely, use `--no-check-certificate'.
```

File: testing/phoenix5/Dockerfile (hunk 1 above)
Comment (wendigo) [reply testing/phoenix5/Dockerfile:35]: Alternative: https://github.com/trinodb/docker-images/pull/188

**Pull request closed without merging**